src/Microsoft.PowerShell.Commands.Utility/commands/utility/DebugRunspaceCommand.cs

```diff
@@ -101,19 +101,11 @@ public Guid InstanceId
         }
```

Comment:
@KirkMunro, your last commit had 2 failures in `PowerShell-CI-windows`
PSModulePath changes interacting with other PowerShell processes.System32 module path prepended to PSModulePath.Allows PowerShell subprocesses to call core modules
```powershell
Expected $null, but got @(Get-ChildItem : The 'Get-ChildItem' command was found in the module 'Microsoft.PowerShell.Management', but the module could not be loaded. For more information, run 'Import-Module Microsoft.PowerShell.Management'., At line:1 char:1, + Get-ChildItem, + ~~~~~~~~~~~~~, + CategoryInfo          : ObjectNotFound: (Get-ChildItem:String) [], CommandNotFoundException, + FullyQualifiedErrorId : CouldNotAutoloadMatchingModule,  ).
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Core\CompatiblePSEditions.Module.Tests.ps1: line 353
353:             $errors | Should -Be $null
```
Default enablement of Experimental Features.On preview builds, Experimental Features are enabled
```powershell
Expected: Feature PSCommandNotFoundSuggestion to be Enabled
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\ExperimentalFeature\Get-ExperimentalFeature.Tests.ps1: line 176
176:             $expFeature.Enabled | Should -BeEnabled -Name $expFeature.Name
```